package/gluon-mesh-vpn-tunneldigger/files/usr/bin/tunneldigger-watchdog

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/lua
+
+function restartTunneldigger()
+	io.popen('logger -t tunneldigger-watchdog "Restarting Tunneldigger."')
+	os.execute("/etc/init.d/tunneldigger restart")
+end
+
+function readPidFile()
+	local pidFile = io.open("/var/run/tunneldigger.mesh-vpn.pid", "r")
+	local returnPid = "0"
+	if pidFile~=nil then 
+		returnPid = pidFile:read("*l") 
+		io.close(pidFile)
+	end
+	return returnPid
+end
+
+function getNeighbourCount()
```

Comment:
The code looks more like a `hasTunneldigerNeighbours` than a request for a count. You return one of (1, nil) curently but (true, false) would be more appropriate then.

Reply:
True, we had first counted the neighbours, before we noticed that that wastes cpu cycles.

nil and false is the same in lua, just like 1 (or any number) and true. I'll change it anyway.